xcube/core/dsio.py

```diff
@@ -23,22 +23,30 @@
 import shutil
 import warnings
 from abc import ABCMeta, abstractmethod
-from typing import Any, Callable, Dict, Iterable, List, Optional, Set, Tuple, Union, Mapping
+from typing import Any, Callable, Dict, Iterable, List, Optional, Set, Tuple, \
+    Union, Mapping
 
 import botocore.exceptions
 import pandas as pd
 import s3fs
 import urllib3.util
 import xarray as xr
 import zarr
+from deprecated import deprecated
 
 from xcube.constants import EXTENSION_POINT_DATASET_IOS
-from xcube.constants import FORMAT_NAME_CSV, FORMAT_NAME_MEM, FORMAT_NAME_NETCDF4, FORMAT_NAME_ZARR
-from xcube.core.timeslice import append_time_slice, insert_time_slice, replace_time_slice
+from xcube.constants import FORMAT_NAME_CSV, FORMAT_NAME_MEM, \
+    FORMAT_NAME_NETCDF4, FORMAT_NAME_ZARR
+from xcube.core.timeslice import append_time_slice, insert_time_slice, \
+    replace_time_slice
 from xcube.core.verify import assert_cube
 from xcube.util.plugin import ExtensionComponent, get_extension_registry
 
+_DEPRECATION_REASON = 'use xcube.core.store API instead'
```

Comment:
That's not a reason.
```suggestion
_DEPRECATION_REASON = 'Functionality is redundant. Use xcube.core.store API instead.'
```